app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -87,6 +87,21 @@
                     app:layout_constraintTop_toTopOf="parent"
                     tools:src="@drawable/ic_nova_group_1" />
 
+                <!--<Button-->
```

Comment:
Delete this if it's not used

Reply:
Sorry about this, I had commented the code out for a previous method